docs/metrics.md

```diff
@@ -46,3 +48,24 @@ When you are deploying the build operator in your Kubernetes cluster, you need t
     value: "30,60,90,120,180,240,300,360,420,480"
 [...]
 ```
+
+## Configuration of histogram labels
+
+As the amount of buckets and labels has a direct impact on the number of Prometheus time series, you can disable labels that you do not need using the `PROMETHEUS_HISTOGRAM_DISABLED_LABELS` environment variable. Use a comma-separated value to disable multiple labels. For example:
```

Comment:
`disable` or `enable`, I am not sure which way is better.

But I think enable is more flexible than disable.

Disable is disable from buildstrategy or namespace, I don't know if the operator can understand what options they can choose.

Enable, can we enable more than these two? If possible, the operator can only enable buildstrategy, or enable all, like buildstrategy, namespace and buildrun_name, etc...

Reply:
I asked myself the same @zhangtbj and it mostly only matters once we extend the list of supported labels:

- If we have an environment variable that allows to enable labels, then new labels will not be active by default because the user will have to enable it
- If we have an environment variable that allows to disable labels, then new labels will be active by default assuming the user cannot have disabled non-existing labels upfront

I honestly do not know what is better.

What I will definitely NOT do in this PR is to allow to enable more labels. This PR is to allow customization of the existing labels and not to extend our metrics solution.

Reply:
So just for my understanding, we define the supported labels in the code, right? There is currently no way for an operator of `build` to add another label, because it is hard-coded. And we know that adding more labels increases the number of metrics.

With that in mind, I would opt for an _enable_ model, where we define the labels that could be used and we define that by default this and that label are enabled and the others are disabled. An operator can then -- depending on the system -- decide to enable additional labels.

Reply:
I changed to the enabled approach.